packages/now-client/src/check-deployment-status.ts

```diff
@@ -89,14 +97,29 @@ export async function* checkDeploymentStatus(
     }
 
     if (isAliasError(deploymentUpdate)) {
-      return yield { type: 'error', payload: deploymentUpdate.aliasError };
+      return yield {
+        type: 'error',
+        payload: {
+          message:
+            deploymentUpdate.aliasError ||
```

Comment:
This should be more like this, since the aliasError property is a code + message
```
{
   ...deploymentUpdate.aliasError,
  message: deploymentUpdate.aliasError.message || 'An error occurred while assigning a domain to your deployment.',
}
```